### PR TITLE
Chief1.2 upgrade

### PR DESCRIPTION
**Pull request by KentonPrescott**
This PR adds support for [DS-Chief1.2](https://forum.makerdao.com/t/dschief-1-2-flash-loan-protection-for-maker-governance/5115). `chief-keeper` functionality remains unchanged, but an explicit `launch_chief()` step was required before unit tests could be executed. 

Was given a moment to refactor the unit tests and consolidated some generalized functions in [pymaker](https://github.com/makerdao/pymaker/blob/master/tests/test_governance.py)

@jeannettemcd Once this PR is merged, the `chief-keeper` will need to be re-launched in our production envs.

### CODE REVIEW

File: tests/test_database.py [hunk 1]
```diff
@@ -27,13 +27,11 @@
...
 
-from pymaker import Address
+from pymaker import Address, Contract, Transact
 from pymaker.deployment import DssDeployment
 from pymaker.numeric import Wad
 
-from tests.test_dss import mint_mkr
-
-
+from tests.test_governance import mint_approve_lock, launch_chief
```
Comment (EdNoepel) [inline tests/test_database.py:34]: Is there supposed to be a `test_governance.py`?

Comment (EdNoepel) [reply tests/test_database.py:34]: Ah, I see...it's in `pymaker`.

**Review by EdNoepel — APPROVED** [review]
GG

**Review by godsflaw — COMMENTED** [review]
LGTM

**Review by godsflaw — APPROVED** [review]

**Pull request merged**